lib/charms/vault_k8s/v0/vault_kv.py

```diff
@@ -184,7 +186,7 @@ class AppVaultKvRequirerSchema(BaseModel):
 class UnitVaultKvRequirerSchema(BaseModel):
     """Unit schema of the requirer side of the vault-kv interface."""
 
-    egress_subnet: str = Field(description="Egress subnet to use, in CIDR notation.")
+    egress_subnets: str = Field(description="Egress subnet to use, in CIDR notation.")
```

Comment:
This is a change to the interface and should come with a change here as well:
- https://github.com/canonical/charm-relation-interfaces/tree/main/interfaces/vault_kv/v0

Reply:
Good point. [PR](https://github.com/canonical/charm-relation-interfaces/pull/163) created.